Altis_Life.Altis/stringtable.xml

```diff
@@ -7975,7 +7975,7 @@
             <Spanish>Controla si los jugadores tienen nombres (tags) sobre sus cabezas.</Spanish>
             <Russian></Russian>
             <German>Regelt, ob Spieler Namensschilder über dem Kopf haben.</German>
-            <French>Contrôle si les joueurs auront ou non leur noms au dessus de leur tête</French>
+            <French>Contrôler si les joueurs auront ou non leur noms au dessus de leur tête</French>
```

Comment:
Two possibilities here, but the second one fits the best.

- "leurs noms" : (in full plural, with 2 "s")
- "leur nom" (singular)